editor/plugins/script_text_editor.cpp

```diff
@@ -1447,7 +1471,7 @@ void ScriptTextEditor::_color_changed(const Color &p_color) {
 	code_editor->get_text_edit()->set_line(color_line, new_line);
 }
 
-void ScriptTextEditor::_make_context_menu(bool p_selection, bool p_color) {
+void ScriptTextEditor::_make_context_menu(bool p_selection, bool p_color, int p_fold_state) {
```

Comment:
Would be better if this was a bool, as it can't be half folded.

Reply:
Agreed.